.travis.yml

```diff
@@ -303,16 +310,17 @@ script:
  - $TRAVIS_BUILD_DIR/.travis/script-$TRAVIS_OS_NAME.sh
  - $TRAVIS_BUILD_DIR/.travis/test.sh
 # only on osx for now
- - echo $DO_DEPLOY
- - if [[ ! -z "$DO_DEPLOY" ]]; then echo hi; fi
- - if [[ -z "$DO_DEPLOY" ]]; then echo hi; fi
- - if [[ ! -z "$DO_DEPLOY" ]]; then $TRAVIS_BUILD_DIR/.travis/package-$TRAVIS_OS_NAME.sh; fi
+ - echo $DEPLOY_AWS_PREFIX
+ - echo $DEPLOY_GITHUB_SUFFIX
+ - if [[ ! -z "$DEPLOY_AWS_PREFIX" ]] || [[ ! -z "$DEPLOY_GITHUB_SUFFIX" ]]; then $TRAVIS_BUILD_DIR/.travis/package-$TRAVIS_OS_NAME.sh; fi
 
 before_deploy:
  # required for github releases
  - export BUILD_PREFIX=$TRAVIS_REPO_SLUG/$TRAVIS_OS_NAME
+ - export ARTIFACT_AWS_NAME = $DEPLOY_AWS_PREFIX$TRAVIS_COMMIT
+ - export ARTIFACT_GH_NAME = $SuperCollider-$VERSION_TO_BUILD$DEPLOY_GITHUB_SUFFIX
```

Comment:
i don't think that `$` before SuperCollider should be there? also, when putting several environment variables right next to each other, using `${ABC}` instead of `$ABC` can improve readability and provide a bit of future proofing. i think i've even seen the recommendation to always use this notation.

Reply:
also also, the names "S3_ARTIFACT_NAME" and "GITHUB_ARTIFACT_NAME" would stylistically match better with the naming throughout

Reply:
also match style by removing the spaces around `=`s

Reply:
ah, good catch with `$SuperCollider`, thanks! also 👍  for all other comments